app/modules/site_settings/resources.py

```diff
@@ -535,7 +535,20 @@ def get(self):
         if is_module_enabled('users'):
             from app.modules.users.models import User
 
-            response['num_users'] = User.query_search().count()
+            num_internal_users = User.query.filter(
+                User.static_roles.op('&')(User.StaticRoles.INTERNAL.mask) > 1
```

Comment:
Something I have bee wondering for a while is how to do an sqlachemy query filtering on user roles.
bitwise & against the mask actually worked. 
The comparison could have been against the mask but anythning that is non 0 matches.